packaging/centos7/cloud-management.service

```diff
@@ -24,6 +24,7 @@ Description=CloudStack Management Server
 After=syslog.target network.target
 
 [Service]
+UMask=0022
```

Comment:
Man, I did not know that. That is why I asked.
I just asked because it was something I did not understand, and after the clarification, I was going to give the LGTM.

Just because one has not read the systemd manual, it does not mean that person is stupid.  I doubt you have even read it too. You might have just consulted it when needed, which is fine.

Do you even know the meaning of "facepalm" or other expressions you use? You are being pretty rude without reason.
But anyways.